editor/src/components/editor/global-shortcuts.tsx

```diff
@@ -954,3 +975,55 @@ function addCreateHoverInteractionActionToSwitchModeAction(
       ]
     : [switchModeAction]
 }
+
+function detectBestWrapperElement(
+  metadata: ElementInstanceMetadataMap,
+  elementPath: ElementPath,
+  makeUid: () => string,
+): WrapInView['whatToWrapWith'] {
+  const element = MetadataUtils.findElementByElementPath(metadata, elementPath)
+  if (
+    element == null ||
+    element.specialSizeMeasurements.parentFlexDirection == null ||
+    element.specialSizeMeasurements.parentLayoutSystem !== 'flex'
+  ) {
+    return 'default-empty-div'
+  }
+
+  const uid = makeUid()
+
+  // pragmatismus maximus, REVIEW: is there a better way to do this?
```

Comment:
Not really. You could create the `gap` upfront as either a value or `undefined` and then shove it straight into the `style` inline. Other than that I'd say we've never really used `Record`, but rather stuck with standard objects for things like this where it doesn't really matter. I have no strong preference either way though.

Reply:
Ok turns out I'm a liar and we have used `Record` in some places

Reply:
> You could create the gap upfront as either a value or undefined and then shove it straight into the style inline.
sounds easy enough to avoid mutation!

also, I think the style object could by typed as 
`const style: CSSProperties = {` !

Reply:
I totally forgot that that type exists! Good catch!

Reply:
thanks! I changed the type to `CSSProperties` in https://github.com/concrete-utopia/utopia/pull/3209/commits/a841cca84ff9c790ce1113f96f855652b345e36f